modules/context/repo.go

```diff
@@ -68,6 +72,36 @@ func (r *Repository) HasAccess() bool {
 	return r.AccessMode >= models.ACCESS_MODE_READ
 }
 
+// GetEditorconfig returns the .editorconfig definition if found in the
+// HEAD of the default repo branch. If any error happens (like if the file
+// doesn't exists) it returns nil
+func (r *Repository) GetEditorconfig() *editorconfig.Editorconfig {
+	if r.editorConfig != nil {
+		return r.editorConfig
+	}
+
+	commit, err := r.GitRepo.GetBranchCommit(r.Repository.DefaultBranch)
```

Comment:
I updated with a little refactoring.

I am now returning the errors for future use, but ignoring them in this case, because we don't want to break anything if the file doesn't exists/can't be parsed.

Maybe your idea would be printing them in the log?